cartesian_velocity_controller:
    type: events_shaping_controller_control/CartesianVelocityController
    arm_id: panda

    # Reasonably stiff robot
    # joint_internal_controller_impedances: [6000.0, 6000.0, 6000.0, 6000.0, 10000.0, 10000.0, 6000.0]

    # Max stiffness for robot
    joint_internal_controller_impedances: [14000.0, 14000.0, 14000.0, 14000.0, 14000.0, 14000.0, 14000.0]


task_priority_joint_velocity_controller:
    type: events_shaping_controller_control/TPJointVelocityController
    arm_id: panda

    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7

    # Default
    # joint_internal_controller_impedances: [1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0]
    
    # Reasonably stiff robot
    joint_internal_controller_impedances: [6000.0, 6000.0, 6000.0, 6000.0, 10000.0, 10000.0, 6000.0]

    # Max stiffness for robot
    # joint_internal_controller_impedances: [14000.0, 14000.0, 14000.0, 14000.0, 14000.0, 14000.0, 14000.0]


franka_state_controller:
  type: franka_control/FrankaStateController
  publish_rate: 30  # [Hz]
  joint_names:
    - panda_joint1
    - panda_joint2
    - panda_joint3
    - panda_joint4
    - panda_joint5
    - panda_joint6
    - panda_joint7
  arm_id: panda